constants.py

```diff
@@ -49,7 +49,8 @@
 SCALER_CONFIG_DEFAULT_SERVER_PRIVATE_IP = '10.0.0.4'
 SCALER_CONFIG_DEFAULT_SERVER_PORT = '33450'
 
-# Platform specific constants
+# Platform specific constants, respecting the EC2 Image Builder requirement of sentence casing
```

Comment:
ah sorry, just noticed we don't have a linux platform const so the plural here makes it sound the the following consts for generator and cache folder also need to be sentence case. 

Suggest saying instead: "**Platform constant**, respecting..."